adhocracy-plus/assets/scss/components/_blocks.scss

```diff
@@ -167,6 +141,8 @@ $block-padding-md: 7.5 * $padding;
     @media screen and (min-width: $breakpoint-xxl) {
         margin-left: 0;
         margin-right: 0;
+        padding-left: 13px; // hack to work with wrapper on larger screens
+        padding-right: 13px;
```

Comment:
ensures the colours background block on extra large screens is aligned with the rest of the homepage wrapper, no weird border